### DISCUSSION TITLE
합의 알고리즘

### DISCUSSION THREAD

**Discussion by k930503** [barreleye-labs/barreleye#19] [Q&A] [answered]
슬슬 합의 알고리즘을 어떻게 구현할 건지 생각해보자.

**Comment by usiyoung** [comment, marked as answer]
> 슬슬 합의 알고리즘을 어떻게 구현할 건지 생각해보자.

POW를 그대로 구현해보거나 POS를 아주 간단한 수준으로 구현해보자!

**Reply by k930503** [reply]
블록 보상 구현하고 POS로 해보자 그럼
